cart/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from cart.views import CartView
```

Comment:
```suggestion
from django.urls import path

from cart.views import CartView
```
모듈 import 컨벤션 수정해주세요~